documentation/synthetic_tokens/explainer.md

```diff
@@ -104,6 +106,13 @@ Here are three ways in which a liquidation can be resolved:
   - The token sponsor will receive any remaining collateral and a reward for the improper liquidation.
     ![](st_liquidation_3.png)
 
+A table summarizing these payouts is below:
+| | Token Sponsor | Liquidator | Disputer |
+|----------------------------------------|----------------------------------------------------------------------------|----------------------------------------------------------------|-------------------------------|
+| Liquidation was not disputed | 0 | Token Sponsor's collateral + Liquidator bond | 0 |
+| Token Sponsor was over-collateralized | Token Sponsor's collateral - Value of tokens + Improper liquidation reward | Value of tokens - Dispute reward - Improper liquidation reward | Dispute bond + Dispute reward |
+| Token Sponsor was under-collateralized | 0 | Token Sponsor's collateral + Dispute bond + Liquidator bond | 0 |
```

Comment:
why does the disputer earn the liquidator bond here and not above? I thought the liquidator bond was to pay for final fees.

Reply:
Good catch - we can fix this.

Reply:
Tables are very difficult to read in raw markdown. If you look at the rich rendering, the disputer never earns the liquidator bond.

<img width="943" alt="Screen Shot 2020-05-08 at 11 48 27 AM" src="https://user-images.githubusercontent.com/11791551/81423064-e7600780-9121-11ea-8110-1b81b083bada.png">

Reply:
Thanks, Matt. 
Row 2 and 3 both indicate that disputes occurred though. Why does the liquiadtor get the liquidator bond returned in row 3, but not row 2?

Reply:
Because in row 2, the liquidator bond pays the final fee (since they were wrong). In row 3, they were correct, so their bond was returned.